README.md

```diff
@@ -42,22 +44,16 @@ docker run -p 3000:3000 -v $PWD:/usr/src/app korerorero-mouth-shapes:latest
 
 ## Installation
 
-Cmake is an external dependency, the OSX version is available here: <https://cmake.org/download/>
-
-This code was tested against `cmake-3.17.0-rc2-Darwin-x86_64.dmg`
+Use the docker instructions above. It's a whole thing to set up this environment on your local machine.
 
-```bash
-nvm use
-cp .env.example .env
-npm i
-```
+Visit <http://localhost:3000/> and you should see the Rhubarb version installed.
```

Comment:
Probably just worth removing this section and referring only to installation using Docker

Reply:
Agreed